client/my-sites/site-settings/jetpack-credentials/credentials-configured/index.jsx

```diff
@@ -134,10 +134,14 @@ class CredentialsConfigured extends Component {
 	}
 }
 
-const mapStateToProps = ( state, { siteId } ) => ( {
-	mainCredentials: getJetpackCredentials( state, siteId, 'main' ),
-	isPressable: isSitePressable( state, siteId ),
-} );
+const mapStateToProps = ( state, { siteId } ) => {
+	const { canAutoconfigure, credentials = [] } = getRewindState( state, siteId );
+
+	return {
+		canAutoconfigure: canAutoconfigure || credentials.some( c => c.type === 'auto' ), // eslint-disable-line wpcalypso/redux-no-bound-selectors,max-len
+		mainCredentials: credentials[ 0 ],
```

Comment:
There is no guarantee that the main credentials will be in the 0th index of this array. We should probably do something like `mainCredentials: find( credentials, { role: 'main' } )` instead.

Reply:
sounds good - I wanted to raise this after the PR merges too, because I think for now we can eliminate the credentials array. the reality is that Rewind currently only has/cares about one set of credentials. I think that will further simplify this stuff.